packages/glibc/glibc.spec

```diff
@@ -13,39 +13,17 @@ Source3: ldconfig-service.conf
 # applied and reverted during the build.
 Source99: HACK-only-build-and-install-localedef.patch
 
-# Upstream patches from 2.35 release branch:
+# Upstream patches from 2.36 release branch:
 # ```
-# git checkout origin/release/2.35/master
-# git format-patch glibc-2.35..
+# git checkout origin/release/2.36/master
+# git format-patch glibc-2.36..
 # ```
-Patch0001: 0001-Regenerate-configure.patch
-Patch0002: 0002-posix-Fix-tst-spawn6-terminal-handling-BZ-28853.patch
-Patch0003: 0003-linux-__get_nprocs_sched-do-not-feed-CPU_COUNT_S-wit.patch
-Patch0004: 0004-linux-Fix-missing-__convert_scm_timestamps-BZ-28860.patch
-Patch0005: 0005-Add-BZ-28860-reference-on-NEWS.patch
-Patch0006: 0006-linux-fix-accuracy-of-get_nprocs-and-get_nprocs_conf.patch
-Patch0007: 0007-linux-Use-socket-constants-time64.h-on-tst-socket-ti.patch
-Patch0008: 0008-string-Add-a-testcase-for-wcsncmp-with-SIZE_MAX-BZ-2.patch
-Patch0009: 0009-x86-Fallback-str-wcs-cmp-RTM-in-the-ncmp-overflow-ca.patch
-Patch0010: 0010-x86-Test-wcscmp-RTM-in-the-wcsncmp-overflow-case-BZ-.patch
-Patch0011: 0011-x86-Fix-TEST_NAME-to-make-it-a-string-in-tst-strncmp.patch
-Patch0012: 0012-NEWS-Add-a-bug-fix-entry-for-BZ-28896.patch
-Patch0013: 0013-elf-Check-invalid-hole-in-PT_LOAD-segments-BZ-28838.patch
-Patch0014: 0014-elf-Replace-tst-audit24bmod2.so-with-tst-audit24bmod.patch
-Patch0015: 0015-resolv-Fix-tst-resolv-tests-for-2.35-ABIs-and-later.patch
-Patch0016: 0016-NEWS-Add-a-bug-fix-entry-for-BZ-28688.patch
-Patch0017: 0017-localedef-Update-LC_MONETARY-handling-Bug-28845.patch
-Patch0018: 0018-localedata-Do-not-generate-output-if-warnings-were-p.patch
-Patch0019: 0019-localedef-Handle-symbolic-links-when-generating-loca.patch
-Patch0020: 0020-hppa-Fix-bind-now-audit-BZ-28857.patch
-Patch0021: 0021-Fix-elf-tst-audit2-on-hppa.patch
-Patch0022: 0022-hppa-Fix-swapcontext.patch
-Patch0023: 0023-hppa-Revise-gettext-trampoline-design.patch
-Patch0024: 0024-hppa-Fix-warnings-from-_dl_lookup_address.patch
-Patch0025: 0025-nptl-Fix-cleanups-for-stack-grows-up-BZ-28899.patch
-Patch0026: 0026-io-Add-fsync-call-in-tst-stat.patch
-Patch0027: 0027-nss-Do-not-mention-NSS-test-modules-in-gnu-lib-names.patch
-Patch0028: 0028-nss-Protect-against-errno-changes-in-function-lookup.patch
+Patch001: 0001-stdlib-Suppress-gcc-diagnostic-that-char8_t-is-a-key.patch
```

Comment:
There are 39 patches (or at least that's the last number in the patches' names), but only 6 are applied, is there a reason why the remaining patches are included but not applied? Also there are two patches that modify `NOTES` and `NEWS`, are they required even if they don't make any functional change?

Reply:
Whoops - that's my mistake. There were only six patches when I first added glibc 2.36, then the number grew 😀 I added the patches but forgot to update the spec.

As for the patches modifying `NOTES` and `NEWS` - the patch series is more maintainable if it's just "all patches to 2.36 since release" rather than cherry-picking a subset.